### PR TITLE
Add binary sensor platform to Husqvarna Automower

### PR DESCRIPTION
**Pull request by Thomas55555**
## Proposed change

Add the documentation for the binary sensor platform.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/113248/
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/husqvarna_automower.markdown [hunk 1]
```diff
@@ -80,6 +82,17 @@ The My Home Assistant redirect feature needs to be setup to redirect to your Hom
 
 Once you have enabled the Husqvarna Automower integration, you should see the following entities:
 
+### Binary Sensor
```
Comment (c0ffeeca7) [inline source/_integrations/husqvarna_automower.markdown]:
```suggestion
### Binary sensor
```

File: source/_integrations/husqvarna_automower.markdown [hunk 2]
```diff
@@ -80,6 +82,17 @@ The My Home Assistant redirect feature needs to be setup to redirect to your Hom
...
+
+The integration will create the following binary sensors:
+
+- Battery charging  
+  *The mower is currently charging. Note that the mower will only report the activity of charging if it was its own decision to go home and charge. For example, if it wants to mow but is charging because the battery is low. If the mower is restricted (for example, due to the weekly schedule), it may charge when in the charging station, but the binary sensor will not show.*
```
Comment (c0ffeeca7) [inline source/_integrations/husqvarna_automower.markdown]: :smile: this sounds a bit like a mower with consciousness, with its own will, wants and needs. 
What does it mean:
- *its own decision*
- *it wants to mow*

File: source/_integrations/husqvarna_automower.markdown [hunk 3]
```diff
@@ -80,6 +82,17 @@ The My Home Assistant redirect feature needs to be setup to redirect to your Hom
...
+- Leaving dock  
+  *The mower is currently leaving the charging station and heading out to a starting point.*
+- Returning to dock  
+  *The mower is on its way home to the charging station.*
+
 ### Lawn Mower
```
Comment (c0ffeeca7) [inline source/_integrations/husqvarna_automower.markdown]:
```suggestion
### Lawn mower
```

File: source/_integrations/husqvarna_automower.markdown (hunk 2 above)
Comment (Thomas55555) [reply source/_integrations/husqvarna_automower.markdown]: Lol. This is from the official API documentation.
It just means, that `battery_charging` is only reported, when the mower is in a regular operation cycle (mowing/charging/mowing/...) and is not forced to stay in the dock, by the user.
But I can improve the deception.

Comment (c0ffeeca7) [reply source/_integrations/husqvarna_automower.markdown]: yes, I think a little rephrase would help.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Thomas55555 :+1:

**Pull request merged**